src/main/kotlin/org/jitsi/rtp/rtcp/CompoundRtcpPacket.kt

```diff
@@ -57,3 +62,12 @@ class CompoundRtcpPacket(
 
     override fun clone(): RtcpPacket = CompoundRtcpPacket(cloneBuffer(0), 0, length)
 }
+
+class CompoundRtcpContainedInvalidDataException(
+    compoundRtcpBuf: ByteArray,
+    compoundRtcpOffset: Int,
+    compoundRtcpLength: Int,
+    invalidDataOffset: Int
+) : Exception("Compound RTCP contained invalid data.  Compound RTCP packet data is: " +
+    "${compoundRtcpBuf.toHex(compoundRtcpOffset, compoundRtcpLength)} Invalid data " +
+    "started at offset ${invalidDataOffset - compoundRtcpOffset}")
```

Comment:
Do we want to include information about the original exception that threw, i.e. to detail what about the data was invalid?

Reply:
Sure, I added the reason into it.